docs/resources/kms_key_material.md

```diff
@@ -0,0 +1,93 @@
+---
+subcategory: "Data Encryption Workshop (DEW)"
+---
+
+# huaweicloud_kms_key_material
+
+Manages a KMS key material resource within HuaweiCloud.
+
```

Comment:
-> **NOTE:** Please confirm that the status of the imported key is `penging import`.